deps/rabbitmq_shovel_management/src/rabbit_shovel_mgmt.erl

```diff
@@ -117,6 +109,32 @@ filter_vhost_user(List, _ReqData, #context{user = User = #user{tags = Tags}}) ->
                          VHost     -> lists:member(VHost, VHosts)
                      end].
 
+restart_shovel(VHost, Name, Node) ->
+    rabbit_log:info("Calling rabbit_shovel_util:restart_shovel on Node '~s' for shovel '~s' in vhost '~s'", [Node, Name, VHost]),
+    case rpc:call(Node, rabbit_shovel_util, restart_shovel, [VHost, Name], infinity) of
+        ok -> true;
+        {_, msg} -> rabbit_log:error(msg),
+            false
+    end.
+
+delete_shovel(VHost, Name, Node, Username) ->
+    rabbit_log:info("Calling rabbit_shovel_util:delete_shovel on Node '~s' for shovel '~s' in vhost '~s'", [Node, Name, VHost]),
+    case rpc:call(Node, rabbit_shovel_util, delete_shovel, [VHost, Name, Username], infinity) of
+        ok -> true;
+        {_, msg} -> rabbit_log:error(msg),
+            false
+    end.
+
+get_shovel_node(VHost, Name, ReqData, Context) ->
```

Comment:
Can we add an UT for this function? We should show to the core team that we contribute the high quality code (and maintain the bar). We don't need to rely on existing tests/files, we can always add new test suite, e.g. a new suite we introduced recently [unit_classic_mirrored_queue_throughput_SUITE](https://github.com/rabbitmq/rabbitmq-server/blob/master/deps/rabbit/test/unit_classic_mirrored_queue_throughput_SUITE.erl)

Reply:
hmm i think from the slack conversation & previous commits i see Ut's not added. do you mean a suite for this specific function? or for all functions - im not sure how the convention is

Reply:
Yeah, from the slack conversion, stable tests are welcome :). maybe adding a new `unit_rabbit_shovel_mgmt_SUITE.erl` with a test to test this newly introduced function there. You can ignore other existing functions. Future changes to this shovel module would also have UTs added.